advanced/Scripts/piholeDebug.sh

```diff
@@ -1088,8 +1088,12 @@ show_blacklist() {
     show_db_entries "Blacklist" "SELECT * FROM blacklist" "4 100 7 10 13 50"
 }
 
-show_regexlist() {
-    show_db_entries "Regexlist" "SELECT * FROM regex" "4 100 7 10 13 50"
+show_regexblacklist() {
+    show_db_entries "Regexblacklist" "SELECT * FROM regex_blacklist" "4 100 7 10 13 50"
```

Comment:
It may be clearer to separate "Regex" and the kind of list, e.g. "Regex blacklist".